engine/src/engine.rs

```diff
@@ -1464,12 +1464,25 @@ where
                 // do not pass on these "internal logs" to caller
                 None
             } else {
-                Some(log.into())
+                Some(evm_log_to_result_log(log))
             }
         })
         .collect()
 }
 
+fn evm_log_to_result_log(log: Log) -> ResultLog {
```

Comment:
I had to remove the `From<Log> for ResultLog` implementation when I moved the type definition to `aurora-engine-types` because I didn't want to pull in the whole `evm` dependency just for that small usage and Rust does not allow trait impls on types defined outside the crate.